pippo-session-parent/pippo-session-spymemcached/src/main/java/ro/pippo/session/spymemcached/SpymemcachedSessionDataStorage.java

```diff
@@ -69,7 +69,6 @@ public SessionData get(String sessionId) {
         if (obj == null) {
             return null;
         }
-        this.sessions.touch(sessionId, idleTime);
         SessionData sessionData = (SessionData) obj;
```

Comment:
Because if the value is null, will give an error when doing the casting `(SessionData) null`